### PR TITLE
add updated about info

### PR DESCRIPTION
**Pull request by theyokohamalife**
1. Adds additional information to the About page.
2. Adds Teresa's custom picture.

### Screenshots
Screen Shot 2021-06-28 at 12 57 51 AM (2)
Screen Shot 2021-06-28 at 1 14 27 AM (2)

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ module.exports = {
     "eslint:recommended",
     "plugin:vue/recommended",
     "plugin:json/recommended",
-    "prettier", // Make sure "prettier" is the last element in this list.
+    // "prettier", // Make sure "prettier" is the last element in this list.
```
Comment (ann-kilzer) [inline .eslintrc.js:12]: Can we please not turn this off

**Review by ann-kilzer — CHANGES_REQUESTED** [review]
Let's ensure consistent formatting, otherwise it gets really challenging to build together. Let me know if I can help

File: .eslintrc.js (hunk 1 above)
Comment (theyokohamalife) [reply .eslintrc.js:12]: Sorry I didn't mean to commit this. I had to turn it off or else it wouldn't let me push due to an error in the screenshot I sent.

File: data/contributors.json [hunk 2]
```diff
@@ -1,35 +1,36 @@
...
+    {
+      "name": "Ann Kilzer",
+      "githubHandle": "ann-kilzer",
+      "twitterHandle": "stonecoldkilzer",
+      "role": "Code Admin",
+      "customPic": "ann-kilzer.jpg"
+    },
+    {
+      "name": "Phillip Ermish",
+      "githubHandle": "ermish",
```
Comment (Ermish) [inline data/contributors.json:18]: feel free to add my twitter as well! @philipermish

**Review by ann-kilzer — APPROVED** [review]
LGTM 🍰

**Pull request merged**